.github/workflows/release.yml

```diff
@@ -17,13 +17,31 @@ jobs:
           distribution: temurin
           java-version: 11
           cache: sbt
+
       - name: Get the version
         id: get_version
         run: |
           version=$(echo ${GITHUB_REF/refs\/tags\//} | cut -c 2-)
           echo "VERSION=$version" >> $GITHUB_OUTPUT
         shell: bash
       - run: sbt "compile; lsp-server/assembly"
+
+      - name: Install jq
+        run: sudo apt-get install jq
+
+      - name: Update version in package.json
+        run: |
+          cd plugin-vscode
+          jq --arg version "${{ steps.get_version.outputs.VERSION }}" '.version = $version' package.json > tmp_package.json && mv tmp_package.json package.json
+
+      - name: Package VSCode Extension
+        run: |
+          sbt copyJARToVSCode
+          cd plugin-vscode
+          npm install -g vsce
+          npm install
+          vsce package -o ralph-lsp-${{ steps.get_version.outputs.VERSION }}.vsix
```

Comment:
I couldn't get this to pass with just `vsce package` without getting permission issues. Locally it passes.